startup/trash/999-load.py

```diff
@@ -1,18 +1,3 @@
-#!/usr/bin/env python
-##############################################################################
-#
-# xpdacq            by Billinge Group
-#                   Simon J. L. Billinge sb2896@columbia.edu
-#                   (c) 2016 trustees of Columbia University in the City of
```

Comment:
You can not strip copy right off of files.

Reply:
I don't think we will use any xpdacq/xpdan... in this project.

Reply:
I think it would be OK to remove the _file_ if it's not needed but not OK to strip the copyright off and keep the rest of the file.

Reply:
@danielballan, @mrakitin, @tacaswell  

I like to work in jupyter environment. In the beginning, I was "loading" that file as "%load /xxx/999-load.py.py" into a cell. I just wanted to reduce the length of the cell by a few lines. Nothing else....

I wish I had deleted "trash" folder so that this discussion never happened.